src/utils.cpp

```diff
@@ -904,7 +904,8 @@ void utils::set_common_curl_options(CURL * handle, configcontainer * cfg) {
 		cookie_cache = cfg->get_configvalue("cookie-cache");
 	}
 
-	curl_easy_setopt(handle, CURLOPT_SSL_VERIFYPEER, cfg->get_configvalue_as_bool("ssl-verify"));
+	curl_easy_setopt(handle, CURLOPT_SSL_VERIFYHOST, cfg->get_configvalue_as_bool("ssl-verifhost"));
```

Comment:
I'm also a bit worried about backwards compatibility here. [Docs on `CURLOPT_SSL_VERIFYHOST`](https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYHOST.html) state:

> When the verify value is 1, curl_easy_setopt will return an error and the option value will not be changed. It was previously (in 7.28.0 and earlier) a debug option of some sorts, but it is no longer supported due to frequently leading to programmer mistakes. Future versions will stop returning an error for 1 and just treat 1 and 2 the same. 

[7.28 was released in [ctober 2012](https://curl.haxx.se/download/). Debian Wheezy (oldstable) still [has 7.26](https://packages.debian.org/wheezy/libcurl3).

Let's change this to something like `cfg->get_configvalue_as_bool("ssl-verifyhost") ? 2 : 0`.